src/utils/Avatar/avatar.scss

```diff
@@ -0,0 +1,36 @@
+.rc-avatar {
+  display: block;
+  &.size {
+    &_sm {
+      width: 32px;
+      height: 32px;
+    }
+    &_md {
+      width: 64px;
+      height: 64px;
+    }
+    &_lg {
+      width: 96px;
+      height: 96px;
+    }
+    &_xl {
+      width: 128px;
+      height: 128px;
+    }
+    &_xxl {
+      width: 256px;
+      height: 256px;
+    }
+
+  }
+
+  img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .circle {
+    border-radius: 100%;
+  }
+  
+}
```

Comment:
Add one line here, please.

Reply:
Done at commit [fix in the end line at avatar styles](https://github.com/castromaciel/ui-rollingcode/pull/11/commits/9e2cc89af46867d05ef399483ef198d9dda5062e)